script/network-policies

```diff
@@ -0,0 +1,6 @@
+cf target -s eyfs-dev
+cf add-network-policy eyfs-cms-dev eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+#cf add-network-policy eyfs-cms-review-pr-478 eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+cf target -s eyfs-pre-prod
+cf add-network-policy eyfs-cms-preprod eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+
```

Comment:
I'm not sure what this file has to do with this change - has it slipped in from something else?

Reply:
yes, this is a helper script I added - will remove